EVENT.md

```diff
@@ -1112,9 +1112,33 @@ trying to measure user visits on, the browser can limit the number `destination`
 sites represented by unexpired sources for a source-site.
 
 The browser can place a limit on the number of a source site's unexpired source's
-unique `destination` sites. When an attribution source is registered for a site
-that is not already in the unexpired sources and a source site is at its limit,
-the browser will drop the new source.
+unique `destination` sites. Source registrations will accept an optional field
+`destination_limit` to allow developers to select the behavior when an attribution
+source is registered for a site that is not already registered by any of the
+unexpired sources and a source site is at its limit.
+
+```jsonc
+{
+  ..., // existing fields
+  "destination_limit": {
+    "algorithm": "lifo", // or "priority_fifo", defaults to "lifo" if not present
+    "priority": "[64-bit signed integer]" // defaults to 0 if not present
+  }
+}
+```
+
+The `priority` field is used to prioritize the destinations registered with this
+source with respect to other destinations for the purpose of source deactivation.
+
+The `algorithm` field is used to select the algorithm to determine the final
+`destination` sites that the browser counts. When "lifo" is used, the browser
+will drop the new source if the source site is at its limit. When "priority_fifo"
+is used, the browser will only count the highest prioritized and most recent
```

Comment:
`will only count` is a bit ambiguous: Does it mean that only the destination sites whose priority equals the max are counted?

In general I think these algorithm descriptions could be clarified a bit more, perhaps with examples.

Reply:
Updated the description for clarity.